CHANGELOG.md

```diff
@@ -8,9 +8,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - `ndarray_to_list` method added to `GeneralDataStructureTransporter`
 - `list_to_ndarray` method added to `GeneralDataStructureTransporter` 
+- NN chain initialized
```

Comment:
Change it to 
```
`neural_network_chain.py` chain
```
Try to be more objective in `CHANGELOG.md`.

Reply:
could you please explain it more? I mean being more objective ...